MatchingGame.js

```diff
@@ -2,12 +2,10 @@ const section = document.querySelector("section");
 const playerLivesCount = document.querySelector("span");
 let playerLives = 10;
```

Comment:
I would introduce another constant called `MAX_PLAYER_LIVES` or something and set this equal to that, and then reuse the variable down at the bottom. This way you can manipulate the total number of lives with 1 constant (instead of modifying 2 places)